### PR TITLE
Remove element tag and class

### PR DESCRIPTION
**Pull request by kfarr**

### CODE REVIEW

**Comment by rahulkgupta** [comment]
for my knowledge, what was the purpose behind this change?

File: src/aframe-streetmix-parsers.js [hunk 1]
```diff
@@ -1007,14 +1001,12 @@ function processSegments(
 
   // create and center offset to center the street around global x position of 0
   var streetParentEl = createCenteredStreetElement(segments);
-  streetParentEl.classList.add('street-parent');
```
Comment (vincentfretin) [inline src/aframe-streetmix-parsers.js:1010]: Be careful, this class is checked in https://github.com/3DStreet/3dstreet/blob/2929a6d8b9aa5c1c2b3281fea2eb6659b26929e1/src/editor/components/components/AddLayerPanel/AddLayerPanel.component.jsx#L110-L111

File: src/aframe-streetmix-parsers.js [hunk 2]
```diff
@@ -1726,7 +1722,6 @@ module.exports.processSegments = processSegments;
 function processBuildings(left, right, streetWidth, showGround, length) {
   const buildingElement = document.createElement('a-entity');
   const clonedObjectRadius = 0.45 * length;
-  buildingElement.classList.add('buildings-parent');
```
Comment (vincentfretin) [inline src/aframe-streetmix-parsers.js:1729]: Be careful, this class is checked in https://github.com/3DStreet/3dstreet/blob/2929a6d8b9aa5c1c2b3281fea2eb6659b26929e1/src/editor/components/components/AddLayerPanel/AddLayerPanel.component.jsx#L110-L111

**Comment by kfarr** [comment]
suggestion: simplify this to remove element tag and tackle class later

File: src/aframe-streetmix-parsers.js (hunk 2 above)
Comment (kfarr) [reply src/aframe-streetmix-parsers.js]: ok we'll change this later

File: src/aframe-streetmix-parsers.js (hunk 1 above)
Comment (kfarr) [reply src/aframe-streetmix-parsers.js]: ok we'll change this later

**Review by vincentfretin — APPROVED** [review]

**Pull request merged**